### PR TITLE
Bugfixes

### PR DESCRIPTION
**Pull request by stgraber**

### CODE REVIEW

**Comment by stgraber** [comment]
jenkins: test this please

**Comment by stgraber** [comment]
jenkins: test this please

**Comment by stgraber** [comment]
jenkins: test this please

File: client/operations.go [hunk 1]
```diff
@@ -147,15 +147,20 @@ func (op *Operation) setupListener() error {
 			return
 		}
 
+		// We don't want concurency while processing events
```
Comment (brauner) [inline client/operations.go]: s/concurency/concurrency/

**Pull request merged**